storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/ConsumerTask.java

```diff
@@ -159,7 +159,7 @@ public void run() {
             while (!closing) {
                 maybeWaitForPartitionsAssignment();
 
-                log.info("Polling consumer to receive remote log metadata topic records");
+                log.trace("Polling consumer to receive remote log metadata topic records");
```

Comment:
Logging this line on each consumer poll will flood the log. Change to trace level